src/main/resources/db/migration/V0.5.33__alter-breeding-methods.sql

```diff
@@ -0,0 +1,24 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+insert into breeding_method (name, abbreviation, description, category, genetic_diversity)
+values
+('Natural variant', 'NVT', 'A recognized naturally occurring variant', 'Selection', 'Generative (+)');
+
+update breeding_method set name = 'Seed increase subset cross' where abbreviation = 'MSC';
+update breeding_method set abbreviation = 'UNK', name = 'Unknown increase method' where abbreviation = 'UIM';
+update breeding_method set name = 'Collected (landrace)' where abbreviation = 'LDR';
```

Comment:
The bug references that "Landrace" doesn't work in the file.  Should the entry in bidb be "Landrace" or "Collected (landrace)"? (maybe this is better asked of @shawnyarnes)